mds/factories.py

```diff
@@ -625,19 +624,17 @@ class EventRecord(factory.DjangoModelFactory):
     class Meta:
         model = models.EventRecord
 
-    id = factory.Sequence(str)
```

Comment:
Why do you want to delete this?

Reply:
Yes ! It's not necessary and may try to assign an existing id in some cases